docs/Releases/release100.md

```diff
@@ -13,7 +13,7 @@ Klaw version 1.0.0 is a first major release.
 - Klaw (formerly Kafkawize) Initial Release 1.0.0
 - This release is cloned from main branch of
   <https://github.com/muralibasani/kafkawize>
-- Package renamed to io.aiven.klaw
+- Package renamed to io.aiven.Klaw
```

Comment:
Changing spelling of "Klaw" should not be part of this PR, since it's for the rule "Klaw Core", can you reverse this one?